configs/e2e.conf

```diff
@@ -40,16 +45,18 @@
 --camera_image_width=1280
 --camera_image_height=720
 --simulator_town=1
+# Spawns near a few speed signs
+--simulator_spawn_point_index=108
```

Comment:
Found that this flag allows us to hard-code spawn points in a map. This one spawns near a few traffic signs so it was helpful for testing.

Reply:
Is there a reference guide for the spawn point index? Is it unique per map?

Reply:
The index is unique per map and I'm not aware of any reference guide. This particular index was determined experimentally by printing out the index.

Reply:
As discussed, moved this to a separate conf file.